integration/kubernetes/nginx.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup() {
+	source "${BATS_TEST_DIRNAME}/lib.sh"
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	master=$(hostname)
+	sudo -E kubectl taint nodes "$master" node-role.kubernetes.io/master:NoSchedule-
+}
+
+@test "Verify nginx connectivity between pods" {
+	service_name="nginx"
+	wait_time=20
+	sleep_time=3
+	cmd="sudo -E kubectl get pods | grep $service_name | grep Running"
+	sudo -E kubectl run "$service_name" --image="$service_name" --replicas=2
+	sudo -E kubectl expose deployment "$service_name" --port=80
+	sudo -E kubectl get svc,pod
+	# Wait for nginx service to come up
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
+	sudo -E kubectl run busybox --rm -ti --restart=Never \
```

Comment:
Here we need to actually verify that the connectivity test passed. So this will need to be enhanced somehow